src/main/java/socialite/logic/commands/EditCommand.java

```diff
@@ -96,9 +99,10 @@ private static Person createEditedPerson(Person personToEdit, EditPersonDescript
         Phone updatedPhone = editPersonDescriptor.getPhone().orElse(personToEdit.getPhone());
         Email updatedEmail = editPersonDescriptor.getEmail().orElse(personToEdit.getEmail());
         Address updatedAddress = editPersonDescriptor.getAddress().orElse(personToEdit.getAddress());
+        Twitter updatedTwitter = editPersonDescriptor.getTwitter().orElse(personToEdit.getTwitter());
         Set<Tag> updatedTags = editPersonDescriptor.getTags().orElse(personToEdit.getTags());
 
-        return new Person(updatedName, updatedPhone, updatedEmail, updatedAddress, updatedTags);
+        return new Person(updatedName, updatedPhone, updatedEmail, updatedAddress, updatedTwitter, updatedTags);
```

Comment:
It looks like the other PRs opted to put the social media handles after the tags, while you chose to put the handle before. For consistency, we should decide on one approach. Any thoughts @AY2122S1-CS2103T-F11-4/developers?

Reply:
Yeap, echo on your point, maybe we can add everything in alphabetical order after the tags in `Person` constructor. Maybe we can discuss this further during the Wed meeting.

Reply:
alright i can change it to alphabetical order

Reply:
Changed it to put handle after tags, similar to the other PRs